### PR TITLE
don't redefine mid for reals here

### PR DESCRIPTION
**Pull request by lucaferranti**
## PR description

before, ILA was redefining `mid(x::Real)` here, as this has been introduced in IntervalArithmetic 0.20.5, no need to do it anymore.

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -13,7 +13,7 @@ StaticArrays = "90137ffa-7385-5640-81b9-e52037218182"
 
 [compat]
 CommonSolve = "0.2"
-IntervalArithmetic = "0.20"
+IntervalArithmetic = "0.20.5"
 IntervalConstraintProgramming = "0.12"
 LazySets = "1.47.2"
```
Comment (schillic) [inline Project.toml:18]:
```suggestion
```

File: Project.toml [hunk 2]
```diff
@@ -23,8 +23,4 @@ julia = "1.6"
 
 [extras]
 IntervalConstraintProgramming = "138f1668-1576-5ad7-91b9-7425abbf3153"
```
Comment (schillic) [inline Project.toml:25]:
```suggestion
```

Comment (schillic) [reply Project.toml:25]: and the last line, but I cannot suggest to remove it

File: Project.toml (hunk 1 above)
Comment (schillic) [reply Project.toml:18]: Sorry, that will not help.

**Comment by lucaferranti** [comment]
@schillic the failure here is only on nightly (and for windows only), should we just merge this and leave the investigation to further PR instead of commenting out the tests with ICP?

**Comment by schillic** [comment]
Then I would suggest to allow nightly failures in the build.

File: .github/workflows/CI.yml [hunk 3]
```diff
@@ -10,6 +10,7 @@ jobs:
   test:
     name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }} - ${{ github.event_name }}
     runs-on: ${{ matrix.os }}
+    continue-on-error: ${{ matrix.version == 'nightly' }}
```
Comment (schillic) [inline .github/workflows/CI.yml:13]: Did it work? I am not sure...
```suggestion
    continue-on-error: ${{ matrix.julia-version == 'nightly' }}
```
Not sure, just saw this elsewhere.

**Review by schillic — APPROVED** [review]

File: .github/workflows/CI.yml (hunk 3 above)
Comment (lucaferranti) [reply .github/workflows/CI.yml]: I think in the `matrix.XX` the `XX` should match the name of the variable as defined above, in this case `version` see [here](https://github.com/JuliaIntervals/IntervalLinearAlgebra.jl/blob/main/.github/workflows/CI.yml#L11) . 

My understanding is that if it "works" then you will still see individual failures in the summary in the PR but when you merge the readme badge will be green if failure is only on nightly

Comment (schillic) [reply .github/workflows/CI.yml]: Okay, then better undo it.

**Comment by schillic** [comment]
@lucaferranti, I still get warnings about this conflict. Maybe tag a new release?

**Pull request merged**